scalyr_agent/agent_main.py

```diff
@@ -716,6 +815,22 @@ def __detailed_status(
         if health_check:
             status_format = "json"
 
+        # List of all debug stats that are captured during status report.
+        debug_stats = []
+
+        def print_debug_stats():
+            print(
```

Comment:
Wonder if we should take ``status_format`` command line option value into account here as well and only use machine readable json format in case json is used, otherwise default to more user-friendly format.

WDYT? Or is this an over kill / not needed?

Reply:
I am not sure that it can be more user-friendly with the amount of information that we gather here. I think it has to be readable at least at some level with pretty-printing the json, that we already do.